ray-curator/ray_curator/stages/deduplication/gpu_utils.py

```diff
@@ -1,5 +1,25 @@
+import pynvml
+import ray
+
+
 def align_down_to_256(memory_size: int) -> int:
     """
     Aligns a memory size down to the nearest multiple of 256.
     """
     return (memory_size // 256) * 256
+
+
+def get_device_free_memory() -> int | None:
```

Comment:
I thought we had this function floating around somewhere else in the repo already?